Android/toonDra/.gitignore

```diff
@@ -0,0 +1,46 @@
+# Built application files
+*.apk
+*.ap_
+
+# Files for the ART/Dalvik VM
+*.dex
+
+# Java class files
+*.class
+
+# Generated files
+bin/
+gen/
+out/
+
+# Gradle files
+.gradle/
+build/
+
+# Local configuration file (sdk path, etc)
+local.properties
+
+# Proguard folder generated by Eclipse
+proguard/
+
+# Log Files
+*.log
+
+# Android Studio Navigation editor temp files
+.navigation/
+
+# Android Studio captures folder
+captures/
+
+# Intellij
+*.iml
+.idea/
+
+# Keystore files
+*.jks
+
+# OSX
+.DS_Store
+
+# etc
+!gradle-wrapper.jar
```

Comment:
nope the changes are made with commits stacked.
this will give conflict when the other pull request is made.
use `git commit --amend` and `git push -f`